### DISCUSSION TITLE
Error while pulling code from git on karavan web(deployed on docker)

### DISCUSSION THREAD

**Discussion by MN-Dheeraj** [apache/camel-karavan#1261] [Q&A]
### Ask a question

I created a project and developed my integration with routes etc. I am able to push to the repository that is configured behind.
I could see the files branch of the git repository as well.
I checked out code into my system and made further developments in Visual studio. I pushed this latest code to the repository and tried pulling the latest code into my project on karavan web(docker).
However, I see the following issue while pulling.

```
Index 0 out of bounds for length 0
Error while pulling from git
```

Best Regards,
MN. Dheeraj

### Variant

Web Application

### Container Management (if applicable)

Docker

### Operating System (if applicable)

Windows

### Version

4.4.0

**Comment by nickhristovta** [comment]
I think you need to add a couple of minimum files. application.properties and docker-compose.yaml
